rust/chains/hyperlane-cardano/Cargo.toml

```diff
@@ -15,4 +15,4 @@ hex = "0.4.3"
 tracing = { workspace = true }
 reqwest = { workspace = true }
 tokio = { workspace = true, features = ["macros"] }
-hyperlane-cardano-rpc-rust-client = "1.0.2"
\ No newline at end of file
+hyperlane-cardano-rpc-rust-client = { path = "../../../../hyperlane-cardano/generated-sources/hyperlane-cardano-rpc-rust-client" }
```

Comment:
This is only development. Should publish from `hyperlane-cardano` before reverting and merging this.